pep-0249.txt

```diff
@@ -1297,6 +1295,23 @@ Footnotes
     ``WHERE`` clause, or clearly document a different interpretation
     of the ``.rowcount`` attribute.
 
+.. [10] In Python 2 and earlier versions of this PEP,
+    ``exceptions.StandardError`` was used as the base class for all
+    DB-API exceptions. Since ``StandardError`` was removed in Python 3,
+    database modules targeting Python 3 should use
+    ``exception.Exception`` as base class instead. Both
```

Comment:
Python 3 doesn't have the ``exceptions`` module either!

```
import exceptions
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'exceptions'
```

Reply:
Good point :-)